package.json

```diff
@@ -20,15 +20,24 @@
   "homepage": "https://github.com/team419/comments-module#readme",
   "devDependencies": {
     "@babel/core": "^7.4.4",
+    "@babel/plugin-proposal-class-properties": "^7.4.4",
     "@babel/preset-env": "^7.4.4",
     "@babel/preset-react": "^7.0.0",
     "babel-loader": "^8.0.6",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.13.1",
+    "faker": "^4.1.0",
+    "file-loader": "^3.0.1",
+    "jest": "^24.8.0",
+    "mongoose": "^5.5.10",
     "webpack": "^4.31.0",
     "webpack-cli": "^3.3.2"
   },
   "dependencies": {
+    "axios": "^0.18.0",
     "express": "^4.16.4",
     "react": "^16.8.6",
+    "react-com": "^1.0.6",
```

Comment:
What are you using React-com for?

Reply:
@bill i accidentally miss spelled react dom but ill uninstall it